api-spec/api-pagopa-proxy.yaml

```diff
@@ -649,7 +657,9 @@ definitions:
       - PPT_AUTORIZZAZIONE
       - PPT_CODIFICA_PSP_SCONOSCIUTA
       - PPT_SEMANTICA
+      - PAA_SEMANTICA
```

Comment:
Nitpick: PAA_SEMANTICA should go before PPT_SEMANTICA to reflect docs' description

Reply:
thanks